code/procs/logged_phrases.dm

```diff
@@ -156,19 +156,19 @@ var/global/datum/phrase_log/phrase_log = new
 			phrase = strip_html_tags(phrase)
 		phrase = html_decode(phrase)
 		if(is_sussy(phrase))
-			SEND_GLOBAL_SIGNAL(COMSIG_GLOBAL_SUSSY_PHRASE, "<span class=\"admin\">Sussy word - [key_name(user)] [category]: \"[phrase]\"</span>")
+			SEND_GLOBAL_SIGNAL(COMSIG_GLOBAL_SUSSY_PHRASE, "<span class='admin'>Sussy word - [key_name(user)] [category]: '[phrase]'</span>")
 		#ifdef RP_MODE
 		if(category != "ooc" && category != "looc" && category != "deadsay" && is_ic_sussy(phrase))
-			SEND_GLOBAL_SIGNAL(COMSIG_GLOBAL_SUSSY_PHRASE, "<span class=\"admin\">Low RP word - [key_name(user)] [category]: \"[phrase]\"</span>")
+			SEND_GLOBAL_SIGNAL(COMSIG_GLOBAL_SUSSY_PHRASE, "<span class='admin'>Low RP word - [key_name(user)] [category]: '[phrase]'</span>")
 		#endif
 		if(is_uncool(phrase))
 			var/ircmsg[] = new()
 			ircmsg["key"] = user.key
 			ircmsg["name"] = (user?.real_name) ? stripTextMacros(user.real_name) : "NULL"
-			ircmsg["msg"] = "triggered the uncool word detection: [category]: \"[phrase]\""
+			ircmsg["msg"] = "triggered the uncool word detection: [category]: '[phrase]'"
 			SPAWN(0)
 				ircbot.export("admin", ircmsg)
-			SEND_GLOBAL_SIGNAL(COMSIG_GLOBAL_UNCOOL_PHRASE, "<span class=\"admin\">Uncool word - [key_name(user)] [category]: \"[phrase]\"</span>")
```

Comment:
These re actual quotation marks displayed in a message. Are we sure we want to change that?

Reply:
oh yeah these ones are a mistake